jmx-metrics/src/main/resources/target-systems/solr.groovy

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-def beanSolrCoreSearcherNumDocs = otel.mbean("solr:dom1=core,dom2=*,category=SEARCHER,scope=searcher,name=numDocs")
+def beanSolrCoreSearcherNumDocs = otel.mbeans(["solr:dom1=core,dom2=*,category=SEARCHER,scope=searcher,name=numDocs"])
```

Comment:
The function's overloaded, you don't need to call with a bracketed list, you can just call mbeans with a single string and still resolve the issue that you're encountering.

Reply:
@dehaansa fixed